torch/ao/quantization/qconfig.py

```diff
@@ -244,6 +244,9 @@ def get_default_qconfig(backend='fbgemm', version=0):
         elif backend == 'onednn':
             qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=False),
                               weight=default_per_channel_weight_observer)
+        elif backend == 'x86':
+            qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=True),
```

Comment:
please don't use `reduce_range`, and use `quant_min`/`quant_max` directly

Reply:
> please don't use `reduce_range`, and use `quant_min`/`quant_max` directly

I have a question. What should `quant_min`/`quant_max` be if `reduce_range=True`? I thought they should be 0 and 127, but they are 0 and 255 for fbgemm:
```python
if backend == 'fbgemm':
    qconfig = QConfig(activation=FusedMovingAvgObsFakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
                                                                         quant_min=0,
                                                                         quant_max=255,
                                                                         reduce_range=True),
                      weight=default_fused_per_channel_wt_fake_quant)
```